CreateMassiveWhichReceiveTheSizeFromCommandLine.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class CreateMassiveWhichReceiveTheSizeFromCommandLine {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input the length of the array ");
+        int n = sc.nextInt();
+        if (n < 1) {
+            System.exit(1);
+        }
+        int arr[] = new int[n];
+        switch (n) {
+            case 3:
+                arr = new int[3];
+                break;
+            case 4:
+                arr = new int[4];
+                break;
+            case 5:
+                arr = new int[5];
+                break;
+            case 6:
+                arr = new int[10];
+                break;
+            case 7:
+                arr = new int[10];
+                break;
```

Comment:
You should be able to optimize and shorten the code. Do that.

Reply:
Some changes are done:

Reply:
Changes are done, but they didn't optimize the code at all. The solution works correctly, no objections, but you can still shorten the code a bit more.